corehq/apps/app_manager/templates/app_manager/partials/nav_menu_media.html

```diff
@@ -1,8 +1,8 @@
 {% load i18n %}
 {% load hq_shared_tags %}
-<div id="media_image">
+<div id="{{ qualifier|default_if_none:"" }}media_image">
     <div class="control-group">
-        <label class="control-label" for="menu_image_path">{% trans "Icon" %}</label>
+        <label class="control-label" for="menu_image_path">{% trans ICON_LABEL %}</label>
```

Comment:
That would allow you to go back and remove those parameters [here](https://github.com/dimagi/commcare-hq/pull/5603/files#diff-337a868e26a4d67b3fcf286e8bf87872R244) and [here](https://github.com/dimagi/commcare-hq/pull/5603/files#diff-3277b1bafb8993e99da42df34530cdc3R28)